packages/chart/src/components/PieChart.tsx

```diff
@@ -143,6 +173,7 @@ export default function PieChart() {
           newFilteredData.push(d)
         }
       })
+      debugger
```

Comment:
remove

Reply:
Woops

Reply:
My exclude filters on global search was excluding .tsx files which is why I was not finding this in a search for debugger statements prior to PR.
<img width="594" alt="Screen Shot 2023-02-02 at 11 39 44 AM" src="https://user-images.githubusercontent.com/122304631/216386772-9643e51f-acf8-449c-9e42-2bcea54212ef.png">